eegnb/experiments/auditory_oddball/aMMN.py

```diff
@@ -12,7 +12,7 @@
 
 
 def present(
-    save_fn: None,
+    save_fn: str = None,
```

Comment:
@JohnGriffiths This was likely a typo you introduced in https://github.com/NeuroTechX/eeg-notebooks/pull/80/files

What you wrote means "`save_fn` will be of type `None`", but what you meant was "`save_fn` should have default value `None`".

Reply:
Got it!